core-services/prow/02_config/_plugins.yaml

```diff
@@ -8369,7 +8383,6 @@ plugins:
   - approve
   - assign
   - blunderbuss
-  - bugzilla
```

Comment:
Remove this. You must not disable plugins, the automation will enable them anyway.

Reply:
Ok, thanks!